templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+  <li><a href="#">Home page</a></li>
+  <li><a href="#">Manufacturers</a></li>
+  <li><a href="#">Cars</a></li>
```

Comment:
The link `href="#"` is a placeholder. Consider updating it to point to the actual URL for the Cars page.